doc/install/openmc.rst

```diff
@@ -0,0 +1,42 @@
+.. |DAG-Code| replace:: OpenMC
+..  _OpenMC: https://mit-crpg.github.io/openmc
+
+Installing for use with OpenMC
+==============================
+
+**Note: DagMC can also be built with support for other physics codes while being installed as a dependency for OpenMC.**
+
+This document explains how to install DAGMC for use with OpenMC, assuming you have
+already installed the required `dependencies <dependencies.html>`_.
+
+DAGMC is an optional dependency of OpenMC_. Therefore, the install process for
+this code only generates and installs the DAGMC libraries necessary for linkage
+with OpenMC.
+
+.. include:: get_dagmc.txt
+
+.. include:: configure_dag-code-header.txt
```

Comment:
This should be `configure_dag-code_header.txt` (last hyphen should be an underscore)